<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 89, 156);</string>
  </property>
  <widget class="QPushButton" name="ButtonPrenotazioni">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>175</y>
     <width>300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(172, 171, 172);
color: rgb(230, 235, 239);
border-radius: 15px;
border-color: 1px rgb(230, 235, 239);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/MenuInfermieriPrenotazioniPiccolo.png</normaloff>../../../../MATERIALE/MenuInfermieriPrenotazioniPiccolo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>220</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPazienti">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>175</y>
     <width>300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(172, 171, 172);
color: rgb(230, 235, 239);
border-radius: 15px;
border-color: 1px rgb(230, 235, 239);
text-align: center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/MenuInfermieriPazientiPiccolo.png</normaloff>../../../../MATERIALE/MenuInfermieriPazientiPiccolo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>220</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>8</y>
     <width>571</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 89, 156);
color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Ruolo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>47</y>
     <width>180</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 89, 156);
color: rgba(230, 235, 239,0.7);</string>
   </property>
   <property name="text">
    <string>Ruolo Infermiere</string>
   </property>
  </widget>
  <widget class="QLabel" name="REF">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../../../Menu Medico dark spostato.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLogout">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>406</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../MATERIALE/LogoutBianco.png</normaloff>../../../../MATERIALE/LogoutBianco.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="shadow" stdset="0">
    <string/>
   </property>
   <property name="set" stdset="0">
    <UInt>0</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="LogoNome">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>7</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../MATERIALE/LogoNome.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FrameBianco">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New'; background-color:#2b2b2b;&quot;&gt;&lt;br/&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../MATERIALE/InterfacciaMedico/FrameBianco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>REF</zorder>
  <zorder>FrameBianco</zorder>
  <zorder>ButtonLogout</zorder>
  <zorder>LogoNome</zorder>
  <zorder>Welcome</zorder>
  <zorder>Ruolo</zorder>
  <zorder>ButtonPrenotazioni</zorder>
  <zorder>ButtonPazienti</zorder>
 </widget>
 <resources>
  <include location="../../../../../asa.qrc"/>
 </resources>
 <connections/>
</ui>
